### PR TITLE
Adding a team member to the website

### PR DESCRIPTION
**Pull request by Hasala123**
Added a new section for team members in the site for a GCI task.
Had to remove content in CNAME folder so that i can host it as well. (can't host many site with same CNAME)

### CODE REVIEW

**Review by liveHarshit — CHANGES_REQUESTED** [review]
?

**Review by nikit19 — CHANGES_REQUESTED** [review]
If you remove the CNAME then on what URL will the site open? Please don't do that. Also see how other team members have been added and don't add your own details there

**Comment by mariobehling** [comment]
Add a link to a test page.

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-fossasia.org
\ No newline at end of file
+
```
Comment (abishekvashok) [inline CNAME:1]: Please don't remove this file. It's required for GitHub to identify the domain

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Intend code + add an existing member

**Comment by mariobehling** [comment]
Closing this due to inactivity.

**Pull request closed without merging**